tests/e2e/test_spyre_embeddings.py

```diff
@@ -49,3 +53,59 @@ def test_output(
                               backend=backend,
                               vllm_results=vllm_results,
                               hf_results=hf_results)
+
+
+@pytest.mark.parametrize("warmup_shapes", [
+    (64, 1),
+    (64, 2),
+    (64, 4),
+])  # (prompt_length/batch_size)
+@pytest.mark.parametrize("backend", get_spyre_backend_list())
+@pytest.mark.parametrize("model", get_spyre_model_list(isEmbeddings=True))
+@pytest.mark.parametrize("vllm_version", ["V0", "V1"])
+def test_scheduling_invariance(
+    model,
+    backend,
+    warmup_shapes,
```

Comment:
I don't see where the shape is actually exported to the environment variables? ie. `VLLM_SPYRE_WARMUP_PROMPT_LENS` and `VLLM_SPYRE_WARMUP_BATCH_SIZES`

Reply:
Yes, it wasn't :D